app/bundles/CoreBundle/Form/Type/PublishDateType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class PublishDateType extends AbstractType
```

Comment:
```suggestion
class DatePickerType extends AbstractType
```
I'm thinking about a more generic name as this type has no connection with publishing.

Reply:
Great idea!